website/templates/project/view_file.mako

```diff
@@ -149,6 +149,7 @@
         file: {
             size: ${size},
             extra: ${extra},
+            sourceUrl: '${source_url | js_str}',
```

Comment:
How about this:

```
sourceUrl: '${(source_url or '') | js_str}',
```

That way, at least you can check for the value to be an empty string if it isn't provided, instead of the string `"None"`. The string representation of the `None` object feeels dirty to me.

I'll leave it up to you, though. Let me know either way and I'll pass it on to Steve.